### DISCUSSION TITLE
ar-scale='fixed' blocking usdz autoconversion on iPad

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4165] [Q&A] [answered]
Tested on iPad Pro 12.9in (3rd gen) and Safari (iOS 15.4)

ar-scale="fixed" is blocking the autoconversion of the model, when ar-scale atribute is removed all is working well, i taught this issue was fixed earlier

**Comment by milesgreen** [comment, marked as answer]
What version of ModelViewer are you using? 

Are you only seeing this issue on the iPad, is it working on other iOS devices?

**Reply by danilokrulj** [reply]
Its working on other iOS devices i tested on ( Iphone 8 and XR ). Im using the latest version of MV.

**Reply by milesgreen** [reply]
With iOS 15.4, there was a known issue (https://github.com/google/model-viewer/issues/3300) with Safari on Apple's side that broke QuickLook when passing a USDZ blob with additional parameters (which is what happens for auto USDZ generation with fixed scaling turned on). Apple fixed this in iOS 15.5.

Only solutions is to upgrade to iOS 15.5+

**Reply by danilokrulj** [reply]
Thank you for the response and for your time! :)
